pkg/kopia/command/parse_command_output.go

```diff
@@ -327,6 +331,96 @@ func parseKopiaProgressLine(line string, matchOnlyFinished bool) (stats *Snapsho
 	}
 }
 
+// RestoreStats is a container for stats parsed from the output of a
+// `kopia restore` command.
+type RestoreStats struct {
+	FilesProcessed  int64
+	SizeProcessedB  int64
+	FilesTotal      int64
+	SizeTotalB      int64
+	ProgressPercent int64
+}
+
+// RestoreStatsFromRestoreOutput parses the output of a `kopia restore`
+// execution for a log of the stats for that execution.
```

Comment:
```suggestion
// and figures out the progress stats for that operation.
```
?

Reply:
I've rephrased it according to my previous suggestion: https://github.com/kanisterio/kanister/pull/2776#discussion_r1607134503